_includes/help/nav_page_top.html

```diff
@@ -0,0 +1,18 @@
+<div class="bg-navy flex sm-show">
+  <div class="container cntnr-wide text-decoration-none flex">
+    <h1 class="white sm-pl0">Help</h1>
```

Comment:
I think this `sm-pl0` can just be a `pl2` since we don't need to specify `sm` since the entire container is hidden on mobile and the `2` just to line it up with the rest of the elements on the page.

![screen shot 2017-05-31 at 1 56 20 pm](https://cloud.githubusercontent.com/assets/1178494/26646241/dff84cee-4608-11e7-839b-6fa1a00db5d9.png)